### ISSUE TITLE
wrong path in manifest.json on windows

### ISSUE THREAD

**Issue by versh23** [symfony/webpack-encore#33]
```
  "build/images\\icon-view.png": "/build/images/icon-view.png",
  "build/images\\logo-big-white.png": "/build/images/logo-big-white.png",
```

with that files JsonManifestVersionStrategy does not work correct

**Comment by weaverryan** [comment]
Thanks for the report @versh23! I'll fix this in #28 so that the keys all use `/` instead of `\\`. Do all of the keys contain a `\\` in them, or only the images?

**Comment by versh23** [comment]
yes, only for images and fonts.

**Comment by weaverryan** [comment]
Definitely will be fixed in #28 - I caught the exact issue and added a test case for it :)
